core/include/drivers/gic.h

```diff
@@ -56,4 +58,11 @@ void gic_init_per_cpu(void);
 
 /* Print GIC state to console */
 void gic_dump_state(void);
+
+/*
+ * Reassign one of the SPIs to normal world and set its priority to
+ * GIC_SPI_PRI_NS_EL1. Ensure that the interrupt is disabled when this
```

Comment:
"Ensure that the interrupt is **currently assigned to secure world and** disabled when this ..."

Reply:
Done